sirepo/pkcli/test_http.py

```diff
@@ -152,8 +152,7 @@ async def _sims_tasks():
                 s.append(
                     _Sim(a, e.name, r.report, r.binary_data_file).create_task(),
                 )
-                # Sleep a bit to not completely flood server
-                await tornado.gen.sleep(1)
+                await _pause_for_server()
```

Comment:
@mkeilman this is strange. This already exists in master. Seems like merging/rebasing problems.

Reply:
nvm I see now the base branch for this is issue/3656-radia-distributed-parallel which doesn't have the change.